remote-codec/remote-codec-server/oidc.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"go.temporal.io/server/common/authorization"
+	"go.temporal.io/server/common/config"
+	"go.temporal.io/server/common/log/tag"
+)
+
+type Provider struct {
+	Issuer   string `json:"issuer"`
+	JWKS_URI string `json:"jwks_uri,omitempty"`
```

Comment:
I know it seems weird to combine these, but still not bet to use an underscore here

Reply:
What would it be? `JWKSURI`?